api/openapi-spec/swagger.json

```diff
@@ -2,7 +2,7 @@
   "swagger": "2.0",
   "info": {
    "title": "Kubernetes",
-   "version": "unversioned"
+   "version": "v1.6.0"
```

Comment:
I'm still a bit worried that this isn't autogenerated in some way. it seems likely to drift out of date.